tests/test_mdnsbridge.py

```diff
@@ -14,75 +14,17 @@
 
 import unittest
 import mock
-
-def route_decorator(path):
-    def decorate_function(f):
-        f.mock_path = path
-        return f
-    return decorate_function
+from mdnsbridge.mdnsbridge import *
 
 class StubWebAPI(object):
     pass
 
+
 class AbortException(Exception):
     def __init__(self, code, *args, **kwargs):
         self.code = code
         super(AbortException, self).__init__(*args, **kwargs)
 
-with mock.patch('nmoscommon.webapi.WebAPI', StubWebAPI):
-    with mock.patch('nmoscommon.webapi.route', side_effect=route_decorator) as route:
-        from mdnsbridge.mdnsbridge import *
```

Comment:
There's a couple of ways round this I think - one is that Flask (well Werkzeug) comes with a test client that lets you start the API (kinda) - see http://flask.pocoo.org/docs/1.0/testing/

The other is that @jamesba is writing test helpers for testing APIs, then you'd just have to mock out self.mdns.get_services to give back some sensible JSON.

But this might be nudging up on "too dumb to fail" since it just passes through some JSON stuff. If you think it's worth testing, maybe open a new issue to pick up another time?